encoders/lendianSteganography.py

```diff
@@ -0,0 +1,89 @@
+# Lendian stegonography system
```

Comment:
Could you add some documentation noting that this only works for sound?

As it stands, I would assume it works on images.